runtime/doc/vim_diff.txt

```diff
@@ -372,6 +372,7 @@ Lua interface (|lua.txt|):
 Commands:
   |:doautocmd| does not warn about "No matching autocommands".
   |:wincmd| accepts a count.
+  |:write| shows a prompt if the file was updated externally.
```

Comment:
This is wrong.

Reply:
You're right,  I changed the message. Not sure if it's the right place to put it though.